dockstore-webservice/src/main/java/io/dockstore/webservice/resources/SourceControlResourceInterface.java

```diff
@@ -76,12 +94,13 @@ default Token refreshBitbucketToken(Token token, HttpClient client, TokenDAO tok
                 long create = tokenDAO.create(token);
                 return tokenDAO.findById(create);
             } else {
-                throw new CustomWebApplicationException("Could not retrieve bitbucket.org token based on code",
+                throw new CustomWebApplicationException("Could not retrieve " + refreshUrl + " token based on code",
```

Comment:
I think it would be better to say `Could not retrieve bitbucket.org token...` rather than `Could not retrieve https://bitbucket.org/site/oauth2/access_token token...`. Didn't test it, but that looks like what will happen from reading the code.

I know you made this generic to handle both BitBucket and Zenodo,  but I think you could do so without listing the specific url.

Reply:
added code to get domain name to use in error message